Classes/Emogrifier.php

```diff
@@ -479,7 +464,11 @@ private function removeImportantAnnotationFromAllInlineStyles(\DOMXPath $xPath)
             $nodesWithStyleAttribute = $this->getAllNodesWithStyleAttribute($xPath);
         } catch (\RuntimeException $e) {
             if ($this->debug) {
-                throw($e);
+                throw new \RuntimeException(
+                    $e->getMessage(),
+                    1509193363,
```

Comment:
We can just rethrow `$e` here, I think.

Reply:
For the same reason as above, I chose to create a new exception object to be able to specific a distinct exception code.